ibis/backends/pandas/dispatch.py

```diff
@@ -4,11 +4,11 @@
 
 from multipledispatch import Dispatcher
 
-import ibis
+import ibis.client
```

Comment:
Why is this change needed?

Reply:
this was not needed but thought it would be cleaner this way and a few other files do it this way as well, I can change this back if you like

Reply:
Can I resolve this conversation?